src/styles/blocks/star.scss

```diff
@@ -0,0 +1,42 @@
+
+.stars {
+  display: flex;
+  align-items: center;
+  padding-top: 16px;
+  justify-content: space-between;
+
+  &__star {
+    background-image: url("../../../../../../projects/layout_product-cards/src/images/star.svg");
```

Comment:
```suggestion
    background-image: url("images/star.svg");
```
Try it

Reply:
Check other similar cases.

Reply:
unfortunatelly, for some reasone it doesn't work this way, I've tried it many times idk why((